### PR TITLE
[WNMGDS-1402] Remove unnecessary nav els

### PR DESCRIPTION
**Pull request by zarahzachz**
## Summary

[Jira ticket](https://jira.cms.gov/browse/WNMGDS-1402)

For vertical nav: Added conditional logic to wrap link list with a `<nav>` only if it's not nested (using already existing `nested` prop).

For header: WIP

## How to test

For vertical nav: Run `yarn storybook` locally, navigate to http://localhost:6006/?path=/story/components-vertical-nav--default-vertical-nav and open developer tools. You should only find one instance of `<nav>` being used.

For header: Run `yarn storybook:healthcare` and navigate to Header component. Note only one `<nav>` in use.

### CODE REVIEW

**Comment by pwolfert** [comment]
@zarahzachz , is this ready for review?

**Comment by zarahzachz** [comment]
> @zarahzachz , is this ready for review?

Not yet - I wanted to chat with a designer about this to make sure it's good.

**Comment by forrestbaer** [comment]
Is this ready for review now?

**Comment by line47** [comment]
@zarahzachz header looks good to me. Compared the storybook to the live GitHub doc site and looked :100:

File: packages/ds-healthcare-gov/src/components/Header/Header.stories.jsx [hunk 1]
```diff
@@ -17,14 +17,14 @@ export default {
     },
     initialLanguage: {
       description:
-        '**This prop has been DEPRECATED.** Do not use. See [internationaliation documentation](https://design.cms.gov/guidelines/i18n/#default-internationalized-content-in-the-design-system)',
+        '**This prop has been DEPRECATED.** Do not use. See [internationalization documentation](https://design.cms.gov/guidelines/i18n/#default-internationalized-content-in-the-design-system)',
```
Comment (pwolfert) [inline packages/ds-healthcare-gov/src/components/Header/Header.stories.jsx:20]: 👌 nice

**Review by pwolfert — APPROVED** [review]
It's great that you were also able to address the same kind of problem in the core nav component that is also used in the doc site. I tested on the doc site as well and verified that there's only one nav.

**Pull request merged**